Docs - Improve SEO (#99)
iotaledger/IOTA-2.0-Research-Specifications: docs/1.1Introduction.md

---
description: These are the research specifications for the IOTA 2.0 protocol. Chapter 1 contains control files. Chapter 2 outlines the general structure of the protocol. Chapter 3 discusses the networking layer. Chapters 4, 5, and 6 include the bulk of the protocol.
image: /img/research-specifications/layers-overview.png
keywords:
- messages
- nodes
- approval weight
- value transfer
- tip selection
- access mana
- layers
- communication
- networks
---

# 1.1 Introduction

:::info

> These specifications are for a research prototype and are a work in progress, thus they may change.  Please refer to the preface for a description of their exact status.

Reviewer: ```suggestion
These specifications are in progress and describe a research prototype. They will change — refer to the preface for updates.
```
